ci/Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM docker.io/library/debian:stable-slim
 
 ### meta ###
-ENV RUST_STABLE_VERSION=1.85.0 \
+ENV RUST_STABLE_VERSION=1.86.0 \
```

Comment:
Updated to latest stable as suggested it may resolve some of the issues requiring nightly previously.